### ISSUE TITLE
Friction finding the asset_id to use in buy/sell and linear LP position creation

### ISSUE THREAD

**Issue by zmanian** [penumbra-zone/penumbra#4834]
So I think this problem emerges because registry.json is only partially integrated into pcli.

You can see a bit of my user journey here: https://www.youtube.com/live/EOPFkgE766I?si=DjZt8b1gYESah_HT&t=1755

If I have $HITMOS in my wallet, I see `transfer/channel-4/SHITMOS` as the asset id in `pcl view balance`.

But if I want to LP the asset , I need to use `transfer/channel-4/factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos`.

Ideally we need the asset_ids from view balance to also work as inputs for constructing LP positions.

Especially until/unless we get a GUI  for more of these interactions.

**Comment by aubrika** [comment]
Agreed that this is not at all ergonomic - we are planning some client upgrades this sprint and will try to include this.

**Comment by zbuc** [comment]
The fix for this in `pcli` should be to make the parsing used in the `tx position order sell/buy` commands `registry.json`-aware
